### PR TITLE
fix(api,ui): remove node run sidebar + remove data from getRuns

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14747.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by yesnault — APPROVED** [review]

File: engine/api/workflow_run.go [hunk 1]
```diff
@@ -52,134 +52,114 @@ func (api *API) getWorkflowsRunsAndNodesIDshandler() service.Handler {
...
+				return sdk.WithStack(sdk.ErrWrongRequest)
+			}
 		}
-	}
-	limitS := r.FormValue("limit")
-	if limitS != "" {
-		limit, errAtoi = strconv.Atoi(limitS)
-		if errAtoi != nil {
-			return sdk.WithStack(sdk.ErrWrongRequest)
+		limitS := r.FormValue("limit")
```
Comment (fsamin) [inline engine/api/workflow_run.go:75]: service.FormInt

File: engine/api/workflow_run.go [hunk 2]
```diff
@@ -52,134 +52,114 @@ func (api *API) getWorkflowsRunsAndNodesIDshandler() service.Handler {
...
+		key := vars["key"]
+		name := vars["permWorkflowName"]
+		route := api.Router.GetRoute("GET", api.getWorkflowRunsHandler, map[string]string{
+			"key":          key,
+			"workflowName": name,
+		})
+		// About pagination: [FR] http://blog.octo.com/designer-une-api-rest/#pagination
+		var limit, offset int
+
+		offsetS := r.FormValue("offset")
```
Comment (fsamin) [inline engine/api/workflow_run.go:67]: service.FormInt

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14759.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by richardlt — APPROVED** [review]

**Pull request merged**